# OrbFeaturesSummary Message

std_msgs/Header header

std_msgs/UInt64 timestampEof
std_msgs/UInt64 timestampLastEof

std_msgs/UInt16 featureCount
std_msgs/UInt16 matchCount
std_msgs/UInt64 computeNs



# struct OrbFeaturesSummary {
#   timestampEof @0 :UInt64;
#   timestampLastEof @1 :UInt64;

#   featureCount @2 :UInt16;
#   matchCount @3 :UInt16;
#   computeNs @4 :UInt64;
# }
